### DISCUSSION TITLE
Does it work with HTTP request body?

### DISCUSSION THREAD

**Discussion by rew1nter** [gabriel-vasile/mimetype#454] [Q&A]
![image](https://github.com/gabriel-vasile/mimetype/assets/64508791/4ebc3e85-4c7e-4eab-ab2b-e9143e7e5fe4)
I'm using it like this on a large number of requests that have about 10 different content types. But mimetype says every request is `text/plain`

**Comment by gabriel-vasile** [comment]
`text/plain` is an indication that the input passed to mimetype was empty.
mimetype returns `text/plain` for empty inputs to match the behaviour of `http.DetectContentType` which in turn is based on https://mimesniff.spec.whatwg.org/
